go.mod

```diff
@@ -2,4 +2,4 @@ module github.com/gatling/frontline-doc
 
 go 1.16
 
-require github.com/gatling/gatling.io-doc-theme v0.0.0-20210506143642-a9eb52edaf4d // indirect
+require github.com/gatling/gatling.io-doc-theme v0.0.0-20210611101149-cd6665074904 // indirect
```

Comment:
Should we put this file in the .gitignore?

Reply:
yep, remove this change for now